jvm-packages/xgboost4j-spark/src/test/scala/ml/dmlc/xgboost4j/scala/spark/PersistenceSuite.scala

```diff
@@ -20,10 +20,9 @@ import java.io.File
 import java.util.Arrays
 
 import scala.io.Source
+import ml.dmlc.xgboost4j.scala.{Classification, DMatrix, Regression}
```

Comment:
why we have many adding of imports but never use them?

Reply:
I checked. We use all of those added imports.
One needs to understand why they are not in this diff

Reply:
ah, I think you moved Classification and Regression to xgboost4j leading to these changes